e2e/test_operator_ha_upgrades/operator_ha_upgrade_test.go

```diff
@@ -270,6 +270,9 @@ var _ = Describe("Operator HA Upgrades", Label("e2e", "pr"), func() {
 			// may get bounced at different times and also because of cluster controller
 			// change, which happens a number of times, during an upgrade).
 			Expect(finalGeneration).To(BeNumerically("<=", initialGeneration+80))
+			// Make sure the cluster has no data loss
+			fdbCluster.GetPrimary().EnsureTeamTrackersAreHealthy()
```

Comment:
Probably you are using "data loss" here with a different meaning?

Reply:
What do you mean? This will check if all team trackers contain all data, if we lost some data because we removed a set of processes that still held data, I would expect that the team trackers report an issue. But like I said we can improve this check if we want, but I'd a first start and better than having no checks at all.

Reply:
I take "data loss" to mean the data loaded by data loader is all present in the key-value store/all shards (by taking replication into account). So checking for data loss will involve checking shard sizes etc, but it is fine to extend this check later (if needed).